h/views/main.py

```diff
@@ -57,6 +57,15 @@ def robots(context, request):
 
 @view_config(route_name='stream')
 def stream(context, request):
+    if request.feature('search_page'):
+        q = request.params.get('q', '').split(':')
+        if len(q) == 2 and q[0] == 'tag':
+            tag = q[1]
+            if ' ' in tag:
+                tag = "'" + tag + "'"
```

Comment:
This works, although I think my personal preference is to show users double quotes, this renders `tag:'foo bar'`.
What do you think?